scripts/transform.sh

```diff
@@ -24,17 +24,17 @@ createSymlinks() {
 }
 
 runtarget="build"
-run_tests="true"
+run_tests="false"
```

Comment:
can you add a comment why we don't run tests by default?

just so someone doesn't come a month from now and change this back.